protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -224,6 +224,13 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Transfers"};
   }
 
+  // Get transfer by ID
+  //
+  // Get a specific transfer by ID
+  rpc GetTransferByID(GetTransferByIDRequest) returns (GetTransferByIDResponse) {
```

Comment:
This should either be just `GetTransfer` so that its named the same as all the other `GetSomething` API that retrieve something by their ID, or (which is something more recent thats being done with new API recently, see `ListReferralSets`) adding a `optional string transfer_id` to the `ListTransfersRequest`.

Reply:
I ended up adding this as a separate call because this is only expected to return nothing but the current version of the specific transfer, so the whole `ListTransfers` pagination stuff is just noise, or might suggest that providing the ID would return the the historical transfer data (pending record, final record, etc...). I can change it, though